src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -623,13 +623,14 @@ void ResumeCtrlImpl::AddCommands(ApplicationSharedPtr application,
     const smart_objects::SmartObject& app_commands =
         saved_app[strings::application_commands];
 
-    uint32_t internal_id = 1;
-    for (size_t cmd_num = 0; cmd_num < app_commands.length();
-         ++cmd_num, ++internal_id) {
+    for (size_t cmd_num = 0; cmd_num < app_commands.length(); ++cmd_num) {
```

Comment:
@APipko May we use here `for (auto cmd: app_commands) {  const smart_objects::SmartObject& command = cmd...)`? I mean C++ 11 loops.

Reply:
@AByzhynar if she makes change it according C++11 loops, Livio might treats it like refactoring and will decline this PR.

Reply:
@KVGrygoriev , in this case the fragment of code already changed.
Refactoring in this PR, for example fixing namespaces
https://github.com/smartdevicelink/sdl_core/pull/2515/files#diff-15145cb256e32e52eee68267f0001477R167
And as for me should be moved to separate commit.

Reply:
@AByzhynar I can't use C++11 loop here because app_commands is SmartObject instance and doesn't have methods begin(), end()